hack/devtools/deploy-shared-env.sh

```diff
@@ -42,8 +42,8 @@ deploy_env_dev() {
             "proxyCert=$(base64 -w0 <secrets/proxy.crt)" \
             "proxyClientCert=$(base64 -w0 <secrets/proxy-client.crt)" \
             "proxyDomainNameLabel=$(cut -d. -f2 <<<$PROXY_HOSTNAME)" \
-            "proxyImage=arosvc.azurecr.io/proxy:latest" \
-            "proxyImageAuth=$(jq -r '.auths["arosvc.azurecr.io"].auth' <<<$PULL_SECRET)" \
+            "proxyImage=arointsvc.azurecr.io/proxy:latest" \
+            "proxyImageAuth=$(jq -r '.auths["arointsvc.azurecr.io"].auth' <<<$PULL_SECRET)" \
```

Comment:
has the proxy image been built and published to arointsvc?
is there a follow-up task to delete the proxy image from arosvc, and maybe to rotate all of the development environments?

Reply:
Yes the images are in arointsvc
https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/7293983